fiftyone/operators/operations.py

```
@@ -719,3 +751,30 @@ def browser_download(self, url, filename=None):
+    "green-checkmark",
+    "red-checkmark",
+    "thumbsup",
+    "thumbsdown",
+    "pin",
+    "star",
+    "x",
+    "bookmark",
+}
+
+
+def _validate_selection_style(default, alt):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_validate_selection_style`

Add return type annotation: `None`

(ANN202)